ledger/README.md

```diff
@@ -7,3 +7,59 @@ This package provides two ledger implementations:
 - **Complete Ledger** implements a fast, memory-efficient and reliable ledger. It holds a limited number of recently used states in memory (for speed) and uses write-ahead logs and checkpointing to provide reliability. Under the hood complete ledger uses a collection of MTries(forest). MTrie is a customized in-memory binary Patricia Merkle trie storing payloads at specific storage paths. The payload includes both key-value pair and storage paths are determined by the PathFinder. Forest utilizes unchanged sub-trie sharing between tries to save memory.
 
 - **Partial Ledger** implements the ledger functionality for a limited subset of keys. Partial ledgers are designed to be constructed and verified by a collection of proofs from a complete ledger. The partial ledger uses a partial binary Merkle trie which holds intermediate hash value for the pruned branched and prevents updates to keys that were not part of proofs.
+
+## Definitions
+In this section we provide an overview of some of the concepts. Hence it is highly recommended to checkout [this doc](https://github.com/onflow/flow-go/blob/master/ledger/complete/mtrie/README.md) for the formal and technical definitions in more details.
+
+### binary Merkle tree
```

Comment:
I suggest to use this terminology to be consistent with we define [here](https://github.com/onflow/flow-go/blob/master/ledger/complete/mtrie/README.md#storage-model):
- Merkle trie is the binary radix tree we would like to model. 
- Mtrie (memory-trie) is our implementation of a Merkle trie, with all the optimizations and shortcuts.

```suggestion
### binary Merkle trie
```